packages/zero-client/src/client/zero.test.ts

```diff
@@ -2253,10 +2266,11 @@ test('kvStore option', async () => {
 
     // Firefox is flaky... it takes longer time than Chromium and WebKit.
     // We therefore give it a few times to pass the expectation.
-
-    await tickAFewTimes(clock);
-    await new Promise(resolve => realSetTimeout(resolve, 100));
-    expect(allDataView.data).deep.equal(expectedValue);
+    await tryNTimesAsync(10, async () => {
```

Comment:
Why doesn't `await r.mutate.e.insert({id: 'a', value: 1});` wait until the insertion has been persisted to IDB?  (line 2274)

Reply:
Because w3e write to the LazyStore... We actually have to wait for the persist process for it to become persistent. In Replicache tests we expose persist... That reminds me, we can get the ReplicacheImpl from testing and we can call persist on that.

Reply:
How much work is that? That would be ideal.